ffplayout-api/src/api/routes.rs

```diff
@@ -132,6 +132,7 @@ fn time_before() -> NaiveDateTime {
 struct ProgramItem {
     source: String,
     start: String,
+    title: String,
```

Comment:
I would like to have here a `Option<String>`.

Reply:
I have finished the corrections, sorry for the delay. If there are more corrections, please mention them.